global.json

```diff
@@ -1,5 +1,10 @@
 {
   "projects": [
-    "src", "samples", "tests"
-  ]
+    "src",
+    "samples",
+    "tests"
+  ],
+  "sdk": {
+    "version": "1.0.0-preview2-1-003177"
```

Comment:
do you specify the sdk version explicity in order to get it running on VS 2017? If so, is the VS 2017 mandatory to compile the solution now? If yes, it's not a problem for me, I just want to know